packages/api/endpoints/granules.js

```diff
@@ -12,6 +12,7 @@ const { v4: uuidv4 } = require('uuid');
 const Logger = require('@cumulus/logger');
```

Comment:
As per @Jkovarik , we should add a `// @ts-check` to the top of this file, and make any fixes for the new/edited code